lib/thredded/seed_database.rb

```diff
@@ -51,15 +52,24 @@ def create_topics(count: 26)
 
       @private_topics = FactoryGirl.create_list(
         :private_topic, count,
-        messageboard: messageboard,
         user:         users.sample,
         last_user:    users.sample,
         users:        [user])
     end
 
     def create_posts(count: (0..30))
-      @posts = (topics + private_topics).flat_map do |topic|
-        (count.min + rand(count.max + 1)).times { FactoryGirl.create(:post, postable: topic, messageboard: messageboard, user: users.sample) }
+      @posts = topics.flat_map do |topic|
+        (count.min + rand(count.max + 1)).times do
+          FactoryGirl.create(:post, postable: topic, messageboard: messageboard, user: users.sample)
+        end
+      end
+    end
+
+    def create_private_posts(count: (0..30))
+      @private_posts = private_topics.flat_map do |topic|
+        (count.min + rand(count.max + 1)).times do
+          FactoryGirl.create(:private_post, postable: topic, user: users.sample)
+        end
       end
```

Comment:
`create_posts` and `create_private_posts` are similar enough that they *could* be merged into a single method but the subtle differences were enough that I thought a little duplication was more beneficial than getting too clever with DRY'ing it up.